e2e/davinci-suites/src/phone-number-field.test.ts

```diff
@@ -59,7 +59,7 @@ test('Login - add email device - authenticate with email device', async ({ page
   await page.getByRole('heading', { name: 'Success' });
   await page.getByRole('button', { name: 'Start over' }).click();
 });
-test('Login - add phone device - authenticate with phone device', async ({ page }) => {
+test.skip('Login - add phone device - authenticate with phone device', async ({ page }) => {
```

Comment:
Did you mean to leave this `skip` in the commit? If so, can you leave a comment as to why it's being skipped?